app/assets/javascripts/discourse/app/components/sidebar/edit-navigation-modal-form/categories-form.hbs

```diff
@@ -50,6 +53,9 @@
                   this.selectedSidebarCategoryIds
                   category.id
                 }}
+                disabled={{(not
+                  (includes this.filteredCategoryIds category.id)
+                )}}
```

Comment:
What if parent category is disabled and after filter, I would like to enable it? Wouldn't it be better to keep it available? UI will be more confusing, but it will give the user a full control

Reply:
I find that it is more confusing to enable the checkbox when the dropdown filtered is scoped to `unselected` for example and the parent category is selected.